nservicebus/upgrades/7to8/dependency-injection.md

```diff
@@ -21,6 +21,13 @@ The following adapter packages will no longer be provided:
 
 Instead of the container adapter packages, use the [NServiceBus.Extensions.Hosting](/nservicebus/hosting/extensions-hosting.md) package or the [externally managed container mode](/nservicebus/dependency-injection/#externally-managed-mode) to use a third party dependency injection container. See the [migrating to the Generic Host](#microsoft-generic-host) or [migrating to externally managed mode](#externally-managed-container-mode) sections for further information.
 
+## Behavior changes
+
+The behavior has been aligned with the expectations of the `Microsoft.Extensions.DependencyInjection` package. The following changes have been introduced:
```

Comment:
is "the behavior changed" the right wording here? This is the behavior of using ServiceCollection, which is what changed.  Could we state this e.g. like this "As defined by Microsoft.Extensions.DependencyInjection, IServiceCollection allows to register services multiple times. In such cases, the last registration wins over previous ones. This means that order of registration is relevant".

Reply:
Well this wording is used in an upgrade guide so I think behavior change is the right word because we are comparing v 7 to v8. If that is taken into account I think I'm saying exactly the same as you are suggestion except I'm not further stressing that

> This means that order of registration is relevant

Do you want me to add this?

Reply:
Yeah I guess it depends a bit on perspective. I'm probably familiar enough to understand the impact of changing to the MS DI containers. It feels a bit that we're documenting the external dependencies behavior but I'm fine with that too in this case (and strictly speaking we have our own ServiceCollection implementation :D )

> Do you want me to add this?

It's probably not that important.